app/helpers/application_helper.rb

```diff
@@ -6,7 +6,7 @@ module ApplicationHelper
   # Figgy resource types that can be created in the UI
   # @return [Array<Resource>]
   def all_works
-    [ScannedResource, MediaResource, ScannedMap, RasterResource, VectorResource, NumismaticIssue]
+    [ScannedResource, MediaResource, ScannedMap, RasterResource, VectorResource, NumismaticIssue, Playlist]
```

Comment:
Do we want to suppress discovery of playlists? I thought we wanted to make them private (so end-users couldn't find them), but allow staff to find them.